app/src/main/kotlin/com/simplemobiletools/keyboard/services/SimpleKeyboardIME.kt

```diff
@@ -52,6 +114,14 @@ class SimpleKeyboardIME : InputMethodService(), OnKeyboardActionListener, Shared
         keyboardView!!.setKeyboard(keyboard!!)
         keyboardView!!.setEditorInfo(currentInputEditorInfo)
         keyboardView!!.mOnKeyboardActionListener = this
+        // Inline Suggestion Views
+        // Inline Suggestion Views
```

Comment:
why duplicate comment? Also avoid writing useless comments please

Reply:
Hi @tibbi : I have removed it, and adjusted it a little bit. I think it's better.